tests/cargo-add.rs

```diff
@@ -1091,3 +1091,27 @@ fn add_typo() {
     )
     .unwrap();
 }
+
+#[test]
+fn adds_sorted_dependencies() {
+    let (_tmpdir, manifest) = clone_out_test("tests/fixtures/add/Cargo.toml.sample");
```

Comment:
Could you add a `Cargo.toml.unsorted` file with some dependencies present, but not in sorted order and use it instead?

Reply:
Great idea. Will do it.

Reply:
Done in 4509000a